### DISCUSSION TITLE
ForBo7 // Salman Naqvi - Stable Diffusion, Summarized

### DISCUSSION THREAD

**Discussion by giscus** [ForBo7/forbo7.github.io#4] [Giscus Comments]
# ForBo7 // Salman Naqvi - Stable Diffusion, Summarized

A concise, high level overview on the mechanisms of stable diffusion.

https://forbo7.github.io/forblog/posts/12_stable_diffusion_summarized.html

**Comment by Max-We** [comment]
Nice, I like the diagrams that you created for your explanations.

By the way, there's a typo in "Autoencoder" under https://forbo7.github.io/forblog/posts/12_stable_diffusion_summarized.html.

**Reply by ForBo7** [reply]
Thanks for pointing out the typo! I've fixed it.

"Autoencdoer" 😄

**Comment by SehajDxstiny** [comment]
why is there a need of scheduler in inference?

**Reply by ForBo7** [reply]
I think it's so that the correct amount of noise is removed in each step. 

It's been a while since I've gone through the process, so I may be incorrect, but the noise that the latent has during inference is added in accordance to a scheduler. So that scheduler is needed to correctly remove an amount of noise.

**Comment by CalvHobbes** [comment]
Excellent article, thank you. Really helps understand the fastai lesson better.

**Reply by ForBo7** [reply]
That's great to know! 😀
